packages/vant/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "allowJs": true,
     "noImplicitThis": true,
-    "types": ["vitest/globals", "vue/jsx"]
+    "types": ["vitest/globals", "vue/jsx", "csstype"]
```

Comment:
Thanks! 

I wonder why do we need to add `csstype` here

Reply:
without it will compile error like 
```node
The inferred type of 'xxx' cannot be named without a reference to '.pnpm/csstype@3.1.3/node_modules/csstype'. This is likely not portable. A type annotation is necessary
```

Reply:
get~